skyportal/handlers/api/comment.py

```diff
@@ -687,6 +750,11 @@ def put(self, associated_resource_type, resource_id, comment_id):
                     action='skyportal/REFRESH_SOURCE_GCN',
                     payload={'obj_internal_key': c.obj.internal_key},
                 )
+            elif isinstance(c, CommentOnEarthquake):  # also update the earthquake
+                self.push_all(
+                    action='skyportal/REFRESH_EARTHQUAKE',
+                    payload={'obj_internal_key': c.obj.internal_key},
```

Comment:
Probably want the earthquake ID here too.

Reply:
yeah.